pkg/tsdb/v3iotsdb.go

```diff
@@ -202,15 +202,18 @@ func (a *V3ioAdapter) DeleteDB(deleteAll bool, ignoreErrors bool, fromTime int64
 
 	partitions := a.partitionMngr.PartsForRange(fromTime, toTime)
 	for _, part := range partitions {
-		a.logger.Info("Delete partition '%s'.", part.GetTablePath())
-		err := utils.DeleteTable(a.logger, a.container, part.GetTablePath(), "", a.cfg.QryWorkers)
-		if err != nil && !ignoreErrors {
-			return errors.Wrapf(err, "Failed to delete partition '%s'.", part.GetTablePath())
-		}
-		// Delete the Directory object
-		err = a.container.Sync.DeleteObject(&v3io.DeleteObjectInput{Path: part.GetTablePath()})
-		if err != nil && !ignoreErrors {
-			return errors.Wrapf(err, "Failed to delete partition object '%s'.", part.GetTablePath())
+		pStart, pEnd := part.GetPartitionRange()
```

Comment:
IMHO, if given time interval does contains partial partitions they should never be deleted and user should get notified about.
Only partitions within given interval should be deleted